roadmap.md

```diff
@@ -60,18 +61,18 @@ Microsoft ships multiple .NET Core distros. It is important that Microsoft can s
 Goals
 -----
 
-- There are high-quality ports for Linux, OS X and Windows.
+- There are high-quality ports for Linux, macOS and Windows.
 - There are high-quality ports for X64, X86 and ARM32.
 - .NET Core can be shipped on a schedule that aligns with ASP.NET and Windows 10 UWP dates and quality (e.g. Beta, RTM) requirements.
```

Comment:
I'm wondering if this line is needed, as worded. Instead, I wondering if we should say that we need to maintain the ability to ship a stable quality build every quarter. Basically, avoid talking about products and instead timeframes.